.github/workflows/draft-release.yml

```diff
@@ -22,7 +22,7 @@ on:
 
 
 env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.PAT_FOR_ACTIONS }}
```

Comment:
This means @kira-syslogng will be the author of the release.
We are working on either using the syslog-ng GH org as an author, or we'll just replace Kira's profile picture with something more "official".

Reply:
Done.